apps/explorer/src/pages/transaction-result/TransactionResult.tsx

```diff
@@ -109,9 +96,10 @@ function FailedToGetTxResults({ id }: { id: string }) {
 const transformTransactionResponse = (
     txObj: SuiTransactionResponse,
     id: string
-): TxnState => {
+): DataType => {
     return {
         ...txObj.certificate,
+        transaction: txObj,
```

Comment:
I started wiring this through because it's needed for detailed gas breakdowns, and I want to move to a model where this computation is done on-demand based on the transaction itself vs in the version passed around.